paper/chapters/conclusion.tex

```diff
@@ -1,4 +1,10 @@
-\todo{Future Work: More complicated programs}
-
-\todo{Avoid flagging programs that are branching, but do the same amount of work in each branch.
-(???) From our test data, it seems like we see vulnerabilities sometimes anyways. Might be dangerous to do this.}
\ No newline at end of file
+We have a lot of ideas for future work that we did not get around to. These include:
+\begin{itemize}
+    \setlength\itemsep{-0.6em}
+    \item Extend OptiFuzz to generate more complicated programs, eg. by reintroducing the division and modulus operators while dealing with division by zero in a meaningful way.
+    \item Increasing the precision of the assembly inspection. One way could be to detect if branches contain the same amount of work and identify them as constant time.
```

Comment:
Vote to kill this guy. Vi laver et testing tool, ikke en control-flow analyzer.

Reply:
Hep